lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -99,4 +99,16 @@ static public function conversionFailedInvalidType($value, $toType, array $possi
             implode(', ', $possibleTypes)
         ));
     }
+
+    static public function conversionFailedSerialization($value, $format, $error, \Exception $previous = null)
```

Comment:
The exception message says `Could not convert database value to Doctrine type` which would be wrong then.